package-overrides/github/angular/bower-material@1.0.0-rc4.json

```diff
@@ -0,0 +1,16 @@
+{
+  "main": "index",
+  "format":"cjs",
+  "registry": "jspm",
+  "dependencies": {
+    "angular": "github:angular/bower-angular@^1.4.7",
+    "angular-animate": "github:angular/bower-angular-animate@^1.4.7",
+    "angular-aria": "github:angular/bower-angular-aria@^1.4.7",
+    "css": "github:systemjs/plugin-css@^0.1.9"
+  },
+  "shim": {
+    "angular-material": {
+      "deps": [ "angular" ]
+    }
+  }
+}
```

Comment:
This override also shouldn't be necessary since it is in the original package.json file itself at - https://github.com/angular/bower-material/blob/master/package.json#L12.